scene_common/Dockerfile

```
@@ -33,12 +33,12 @@ RUN : \
     && rm -rf /var/lib/apt/lists/*
 
 # install common dependencies
-COPY ./scene_common/src /tmp/scene_common
+COPY ./scene_common /tmp/scene_common
 RUN : \
     && cd /tmp/scene_common \
     && pip3 install --no-cache-dir pybind11 \
     && pip3 install --no-cache-dir . \
-    && make -C fast_geometry -j $(nproc) all install \
+    && make -C src/fast_geometry -j $(nproc) all install \
```

Let's rename `/tmp/scene_common/src` to `/tmp/scene_common/scene_common` before the installation, so that it get installed as `scene_common`, not `src`. 'src' is too generic name to let it be propagated by the installer. I think it is the best we can do.